test/indicators/loading-container.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { LoadingContainer } from '../../src/'
+
+
+test('LoadingContainer renders div with id "loading"', () => {
```

Comment:
The name is a little misleading, this is really a test to make sure it renders children.